javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,135 @@
+"use strict"
+let name = '';
+let toDo = [];
+
+console.log(getReply(123));
+console.log(getReply("Hello my name is Benjamin"));
+console.log(getReply('What is my name'));
+console.log(getReply('Add fishing to my todo'));
+getReply('Add singing in the shower to my todo');
+console.log(getReply('Remove fishing from my todo'));
+console.log(getReply('What is on my todo?'));
+console.log(getReply('What day is it today?'));
+console.log(getReply('what is 3 + 2'));
+console.log(getReply('what is 3 - 2'));
+console.log(getReply('what is 3 * 2'));
+console.log(getReply('what is 3 / 2'));
+console.log(getReply('Set a timer for 4 minutes'));
+
+
+
+function getReply(command) {
+  if (typeof command !== 'string') {
+    let response = 'please enter valid command';
+    return response;
+  }
+  if (command.slice(0, 16) === 'Hello my name is') {
+    let newString = command.split(" ");
+    name = newString[newString.length - 1];
```

Comment:
What you did is correct. There is also an alternative like below 
There are string methods like 'startsWith' and 'endsWith'.
For example 
   let newString = command.split(" ");

  if (newString .startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }

Reply:
ok thankyou Anna